components/infobox/wikis/starcraft/infobox_map_custom.lua

```diff
@@ -0,0 +1,110 @@
+---
+-- @Liquipedia
+-- wiki=starcraft
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Template = require('Module:Template')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Map = Lua.import('Module:Infobox/Map', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomMap = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomMap.run(frame)
+	local customMap = Map(frame)
+
+	customMap.createWidgetInjector = CustomMap.createWidgetInjector
+	customMap.getWikiCategories = CustomMap.getWikiCategories
+	customMap.addToLpdb = CustomMap.addToLpdb
+
+	_args = customMap.args
+	return customMap:createInfobox()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local id = _args.id
+
+	Array.appendWith(widgets,
+		Cell{name = 'Tileset', content = {_args.tileset or CustomMap:_tlpdMap(id, 'tileset')}},
+		Cell{name = 'Size', content = {_args.tileset or CustomMap:_getSize(id)}},
+		Cell{name = 'Spawn Positions', content = {CustomMap:_getSpawn(id)}},
+		Cell{name = 'Versions', content = {_args.versions and ('\n' .. _args.versions) or nil}},
```

Comment:
Why the new line here?

Reply:
because else the display breaks the wiki list and displays the first as `*` instead of bullet point